microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -558,48 +544,62 @@ See also:
+* This checkbox is present in Microsoft Edge Canary 134.
+* This checkbox is present in Microsoft Edge Stable 132.
 
 
 <!-- ====================================================================== -->
-## Extension storage in Application panel
+## Floating entry points for the AI assistance panel
 <!-- checkbox has no "(?)" link pointing to this anchor wording -->
+<!-- todo: delete section if/when checkbox removed -->
+<!-- consistently hide or reveal all sections that have the above comment -->
 
-_new as of Microsoft Edge Stable 131_
-
-Displays the **Extension storage** section in the **Application** tool.  The **Extension storage** section displays the data that's stored by extensions that are using the storage extension API.
-
-See also: 
-* [chrome.storage](https://developer.chrome.com/docs/extensions/reference/api/storage/) API Reference in the Chrome Extensions docs.
+Not applicable for Microsoft Edge.  Selecting or clearing this checkbox has no effect.
 
 Status:
-* This checkbox is not present in Microsoft Edge Canary 133.
-* This checkbox is present in Microsoft Edge Stable 131.
+* This checkbox is present in Microsoft Edge Canary 134.
+* This checkbox is present in Microsoft Edge Stable 132.
 
 
 <!-- ====================================================================== -->
-<!-- ## Floating entry points for the AI assistance panel -->
-<!-- non-Edge; checkbox has no effect, being removed -->
+## Performance panel: enable performance insights
+<!-- checkbox has no "(?)" link pointing to this anchor wording -->
 
+_new as of Microsoft Edge Stable 129_
 
-<!-- ====================================================================== -->
-## Performance panel: enable experimental performance insights
+In the **Performance** tool, adds the **Insights** sidebar tab, when viewing a recorded performance profile.  The **Insights** sidebar tab provides useful information about the profile, such as the various **Largest Contentful Paint** phases and the list of **Render blocking requests**.
+
+See also:
+* [Introduction to the Performance tool](../evaluate-performance/index.md)
+
+Status:
+* This checkbox is not present in Microsoft Edge Canary 134.
+* This checkbox is present in Microsoft Edge Stable 132.
+
+
+<!-- Stable 132 has both checkboxes -->
+<!-- ------------------------------ -->
+#### Performance panel: enable experimental performance insights
+<!-- checkbox has no "(?)" link pointing to this anchor wording -->
 
 _new as of Microsoft Edge Canary 133_
 
 Includes experimental insights in the **Insights** side panel in the **Performance** tool.
 
-See also: 
+In the **Performance** tool, adds the **Insights** sidebar tab, when viewing a recorded performance profile.  The **Insights** sidebar tab provides useful information about the profile, such as the various **Largest Contentful Paint** phases and the list of **Render blocking requests**.
+<!-- todo: is desc still correct after adding "experimental" to checkbox label? -->
```

Comment:
Check start of desc, specific to "experimental" version of the checkbox:

"Includes experimental insights in the **Insights** side panel in the **Performance** tool."

Stable 132 has both checkboxes:
* **Performance panel: enable performance insights**
* **Performance panel: enable experimental performance insights** - new

@captainbrosset

Reply:
This experiment doesn't enable the Insights side panel. The previous experiment does. 
This experiment enables other insights within the Insights side panel.

So, users should first enable "Performance panel: enable performance insights", and then, if they want to see even more insights, they can enable "Performance panel: enable experimental performance insights".

These additional insights are experimental, and not shown in the side panel by default, probably because they are still being worked on.

Reply:
result:
```
To use this experiment, first select the **Performance panel: enable performance insights** 
checkbox, and then, if you want to see even more insights, also select the **Performance 
panel: enable experimental performance insights** checkbox.
```